File: platform/sdk/python/examples/cloud/cloud-004_head_bucket.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("head-test")
result = client.headBucket("head-test")
if not result["exists"]:
    sys.exit(1)
print("XJ-OK: cloud-004_head_bucket")
sys.exit(0)

File: platform/sdk/nodejs/examples/cloud/cloud-004_head_bucket.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("head-test");
const result = client.headBucket("head-test");
if (!result.exists) process.exit(1);
console.log("XJ-OK: cloud-004_head_bucket");
process.exit(0);
